docs/emcc.txt

```diff
@@ -87,26 +87,25 @@ Options that are modified or new in *emcc* are listed below:
 
      For more tips on optimizing your code, see Optimizing Code.
 
-"-s OPTION[=VALUE]"
+"-sOPTION[=VALUE]"
    [different OPTIONs affect at different stages, most at link time]
    Emscripten build options. For the available options, see
    src/settings.js.
 
    Note:
 
-     You can prefix boolean options with "NO_" to reverse them. For
-     example, "-s EXIT_RUNTIME=1" is the same as "-s
-     NO_EXIT_RUNTIME=0".
+     If no value is specifed it will default to "1".
 
    Note:
 
-     If no value is specifed it will default to "1".
+     You can prefix boolean options with "NO_" to reverse them. For
```

Comment:
If the preferred usage is `-sFOO=0` rather than `-sNO_FOO` (which I think it should be) then this should also say that, and say explicitly what value this form forces (i.e. "0")

Reply:
I already say `"sEXIT_RUNTIME=0" is the same as "-sNO_EXIT_RUNTIME"` in his sentence ..  doesn't that make it clear that 0 is the value that is implied?  Can you suggest a wording you think would be more clear?

Reply:
oh... yeah I think this is OK, I don't know what I missed before. Let's just make sure we say explicitly here what the preferred way is.